sdk/go/common/workspace/plugins.go

```diff
@@ -262,7 +357,7 @@ func (source *githubSource) GetLatestVersion(
 		"https://%s/repos/%s/%s/releases/latest",
 		source.host, source.organization, source.repository)
 	logging.V(9).Infof("plugin GitHub releases url: %s", releaseURL)
-	req, err := buildHTTPRequest(releaseURL, source.token)
+	req, err := buildHTTPRequest(releaseURL, fmt.Sprintf("token %s", source.token))
```

Comment:
Same comment as above: 
It's a smell that the `token %s` is generated in three different places.
A helper method on githubSource would be helpful here.